packages/ui-components/package.json

```diff
@@ -65,16 +70,10 @@
     "typescript": "catalog:",
     "typescript-eslint": "~8.35.1"
   },
-  "imports": {
```

Comment:
Is this just not compatible?

Reply:
It is, i didn't think it was going to be, I've drafted the PR and am planning to re-add it

Reply:
Can't we just use TypeScript imports then? I thought TypeScript would transform those? If the issue is the built packages no being able to reference these imports...

Reply:
No, they _are_ compatable, i didn't think they were, and only realized they would be at the last minute